File: Spring Boot 2.5.0/src/main/java/com/smaaaak/Portfolio/model/Article.java
package com.smaaaak.Portfolio.model;

import com.fasterxml.jackson.annotation.JsonFormat;
import com.fasterxml.jackson.annotation.JsonManagedReference;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.util.ArrayList;
import java.util.Date;
import java.util.List;

@Entity
@Table(name="articles")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Article {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long idArticle ;

    private String title ;

    private String description ;

    @Column(columnDefinition = "TEXT")
    private String content ;

    @JsonFormat(pattern="yyyy-MM-dd HH:mm:ss")
    private Date dateCreation ;

    private String writerName ;

    @ManyToOne
    @JoinColumn(name = "id_category")
    private Category category ;

    @OneToMany(mappedBy = "article")
    @JsonManagedReference
    private List<Comment> comments = new ArrayList<>() ;

    @OneToMany(cascade = CascadeType.REMOVE)
    private List<Image> images = new ArrayList<>() ;

}

File: Angular 13/Admin Side/src/app/shared/model/Article.ts
import { ICategory } from "./Category";
import { IComment } from "./Comment";
import { IImage } from "./Image";

export interface IArticle {

    idArticle : number ;
    title : string ;
    description : string ;
    content : string ;
    dateCreation : string ;
    writerName : string ;
    category : ICategory ; 
    comments : IComment [] ;
    images : IImage [] ;

}

export interface IArticlePages {

    content : IArticle [] ;
    number : number ;
    totalElements : number ;
    size : number ;
    totalPages : number ;

}